.github/workflows/linux-qe-template.yml

```diff
@@ -0,0 +1,202 @@
+            -e DEBUG='true' \
+            -e SSH_AUTH_SOCK=$(cat ssh_auth_sock) \
+            -v "$(cat ssh_auth_sock):$(cat ssh_auth_sock)" \
+            -v ${PWD}:/data:z \
+            -v ${PWD}/crc:/opt/crc-support/crc:z \
+            quay.io/crc-org/ci-crc-support:v2.0.0-dev-linux crc-support/run.sh \
+             -targetPath "/root/crc-support" \
+             -install 'true' \
+             -aName 'crc' \
+             -freshEnv 'false' \
+             -download 'false' 
+          podman logs -f crc-linux-install-${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+
+      - name: Run CRC ${{inputs.qe-type}} test
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        run: |
+          # load image
+          podman load -i crc-${{inputs.qe-type}}-linux-arm64.tar
+          # run
+          cmd="crc-qe/run.sh -junitFilename crc-${{inputs.qe-type}}-junit.xml -targetFolder crc-qe"
+          if [[ "${{inputs.qe-type}}" == "e2e" ]]; then
+            if [[ "${{inputs.preset}}" == "microshift" ]]; then
+              cmd="${cmd} -e2eTagExpression '@story_microshift'"
+            else
+              cmd="${cmd} -e2eTagExpression '~@minimal && ~@story_microshift'"
+            fi
+          fi
+          podman run --rm -d --privileged --name crc-${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}} \
+            -e TARGET_HOST=$(cat host) \
+            -e TARGET_HOST_USERNAME=$(cat username) \
+            -e TARGET_HOST_KEY_PATH=/data/id_rsa \
+            -e BASTION_HOST_USERNAME=$(cat bastion_username) \
+            -e BASTION_HOST=$(cat bastion_host) \
+            -e TARGET_FOLDER=crc-qe \
+            -e TARGET_RESULTS=results \
+            -e OUTPUT_FOLDER=/data \
+            -e DEBUG=true \
+            -e SSH_AUTH_SOCK=$(cat ssh_auth_sock) \
+            -v "$(cat ssh_auth_sock):$(cat ssh_auth_sock)" \
+            -v $PWD/pull-secret:/opt/crc/pull-secret:z \
+            -v $PWD:/data:z \
+            quay.io/crcont/crc-${{inputs.qe-type}}:gh-linux-arm64 \
+                ${cmd} 
+          podman logs -f crc-${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+
+      - name: Test Report
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        id: test-report
+        uses: mikepenz/action-junit-report@v4
+        if: always() 
+        with:
+          fail_on_failure: true
+          include_passed: true
+          detailed_summary: true
+          require_tests:  true
+          report_paths: '**/*.xml'
+      
+      - name: Upload e2e results
+        working-directory: ${{ env.commit_sha }}-${{inputs.qe-type}}-${{inputs.preset}}
+        uses: actions/upload-artifact@v4
+        if: always()
+        with:
+          name: linux-${{inputs.qe-type}}-${{inputs.preset}}
+          path: |
+            **/*.xml
+            **/*.results
+            **/*.log
+
+      - name: Update status of the PR check
```

Comment:
Can you add also the initial api call to indicate the check is starting with the pending state i.e. 

https://github.com/crc-org/crc/blob/42afbf10ec4e0f8e0153ebeae8d307613722988e/.github/workflows/windows-qe-tpl.yml#L84

Also can you add the arch on the message 

Finished ${{inputs.qe-type}}-${{inputs.preset}} on Linux -> Finished ${{inputs.qe-type}}-${{inputs.preset}} on Linux ARM64

Beyond this LGTM ( also consider to add some more info on the commit message) like using for the typo of test (linux arm64)...

Reply:
^^ @lilyLuLiu please if you can change this before merge